core/src/main/java/hudson/model/UsageStatistics.java

```diff
@@ -190,8 +196,10 @@ public String getStatData() throws IOException {
             }
 
             return new String(Base64.getEncoder().encode(baos.toByteArray()));
-        } catch (GeneralSecurityException e) {
-            throw new Error(e); // impossible
+        } catch (Exception e) { // the exception could be GeneralSecurityException, InvalidParameterException or any other depending on the security provider you have installed
```

Comment:
would this ever be an `Error`?  would we be better off with 
```suggestion
        } catch (Throwable e) { // the exception could be GeneralSecurityException, InvalidParameterException or any other depending on the security provider you have installed
```